### ISSUE TITLE
HubSpot form accessibility needs work

### ISSUE THREAD

**Issue by davidmurdoch** [trufflesuite/trufflesuite.com#1202]
This form is controlled by a hubspot script, so it is a difficult to fix this one.

The main problem is that the error messages and error form fields are not labeled with the appropriate error attributes. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage

**Comment by davidmurdoch** [comment]
This form is what I'm talking about:

![image.png](https://images.zenhubusercontent.com/5b6219c0f38e971f32428c32/a8aa890b-4e50-4ab5-bf32-a33dc187db00)
